docs/docs/API-Reference/api-openai-responses.mdx

```diff
@@ -0,0 +1,520 @@
+---
+title: OpenAI Responses API
+slug: /api-openai-responses
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/Tabs';
+
+Langflow includes an OpenAI-compatible API endpoint at `POST /api/v1/responses`.
+This endpoint allows you to run your flows using OpenAI-compatible request and response formats.
```

Comment:
you're overloading the term responses here. the responses api vs request / response format.